docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -298,7 +304,7 @@ WHERE
 
 ### Query Examples with Assist AI
 
-The query editor also integrates with Teleport Assist, making it easy to generate queries with a human-readable description of the data you want to retrieve.
+The query editor also integrates with Teleport Assist, allowing to generate queries with a human-readable description of the data you want to retrieve.
```

Comment:
This sounded better before IMO. I don't think _allowing to_ is grammatically correct. (allowing _who_ to?)

Reply:
"easy" is tagged as a error now in the linting why I changed it. Will look for some other options. Any suggestions @ptgott ?

Reply:
I think "allowing you to" is fine, as the sentence already indicates that the data is something the reader wants to retrieve.

Reply:
Please see verbiage update